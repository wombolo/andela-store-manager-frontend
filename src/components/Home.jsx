import React, { Component } from 'react';

export default class Home extends Component {
  render() {
    return (
      <div>
        <h1>Vy Store Manager</h1>
        <p>Welcome to Vy Store Manager, navigate to login link on the menu to get started</p>
      </div>
    )
  }
}
