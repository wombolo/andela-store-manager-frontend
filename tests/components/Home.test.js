import 'babel-polyfill'
import React from 'react';
import { shallow } from 'enzyme';

import Home  from '../../src/components/Home.jsx';

let wrapper;

describe('Home Component', () => {
  it('should match snapshot', () => {
    wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
