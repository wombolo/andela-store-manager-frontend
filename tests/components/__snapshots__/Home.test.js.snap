// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component should match snapshot 1`] = `
<div>
  <h1>
    Vy Store Manager
  </h1>
  <p>
    Welcome to Vy Store Manager, navigate to login link on the menu to get started
  </p>
</div>
`;
